core/cloudflow-operator/src/main/scala/cloudflow/operator/action/RunnerActions.scala

```diff
@@ -71,13 +67,10 @@ abstract class RunnerActions[T <: ObjectResource](runner: Runner[T]) {
           val resource     = SparkRunner.resource(deployment, newApp, namespace)
           val patch        = SparkRunner.patch(deployment, newApp, namespace)
           val patchAction  = Action.patch(resource, patch)(SparkRunner.format, SparkRunner.patchFormat, SparkRunner.resourceDefinition)
-          val configAction = Action.update(runner.configResource(deployment, newApp, namespace), runner.configEditor)
-          Seq(configAction, patchAction)
+          val configAction = Action.createOrUpdate(runner.configResource(deployment, newApp, namespace), runner.configEditor)
+          Seq(configAction)
```

Comment:
why is the `patchAction` not needed here? Will it be able to `scale` without it? (I think that I added it for that reason)

Reply:
No scale is not working yet.

Reply:
I'm still working on this issue where updates can happen in the wrong order. this patch could happen after the patch that occurs because of config secret change, meaning it uses the defaults for PodConfig and RuntimeConfig, which wipes out the changes to these (which are translated to CR and resource requirements).

Reply:
But it is a good point, I need to solve this differently.

Reply:
I put this back in.